src/PresentationalComponents/RuleDetails/_RuleDetails.scss

```diff
@@ -63,3 +71,15 @@ hr {
     margin-bottom: var(--pf-global--spacer--md);
   }
 }
+
+@media only screen and (max-width: 768px) {
```

Comment:
does this exist as a pf var? feels like we shouldn't be hardcoding this value 🤔

Reply:
yep
`--pf-global--breakpoint--md | global_breakpoint_md | 768px`

Reply:
So did a little digging on this... and it looks like it isn't possible use `var()` in a media query. https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries (unless we want to use a preprocessor?)

Reply:
oooooo  what about `@media only screen and (min-width: $ins-break--md) `

Reply:
actually you dont neet to do that `$ins-break--md: var(--pf-global--breakpoint--md);` is already defined in
@import '~@redhat-cloud-services/frontend-components-utilities/files/Utilities/_all';

Reply:
Oooo I had the same thought 😂 

I think that approach falls victim to the same issue though. The styles don't seem to apply 🤷‍♂️🤔

Reply:
local definition as well?  🤔 I see the value hardcoded (not the pf var) in a few places.. would hate for us to also do it 😢 will take a look tomorrow 👍

Reply:
great minds 🧠 😆

Reply:
well... s on the radar https://github.com/w3c/csswg-drafts/issues/2627 but yeah not currently possible in the way   would hope